blockchain/chain_sync/src/chain_muxer.rs

```diff
@@ -51,7 +51,7 @@ pub enum ChainMuxerError<C: Consensus> {
     #[error("Tipset range syncer error: {0}")]
     TipsetRangeSyncer(#[from] TipsetRangeSyncerError<C>),
     #[error("Tipset validation error: {0}")]
-    TipsetValidator(#[from] TipsetValidationError),
+    TipsetValidator(#[from] Box<TipsetValidationError>),
```

Comment:
It's slightly peculiar to only box one thing but I think it's fine in this case.